recipes/ckanweb-efs-setup.rb

```diff
@@ -77,13 +77,17 @@
     service "supervisord" do
         action [:stop]
     end
-    if ::File.directory?(var_log_dir) then
-        # transfer existing contents to target directory
-        execute "Move existing #{service_name} logs to extra EBS volume" do
-            command "mv -n #{var_log_dir}/* #{real_log_dir}/; find #{var_log_dir} -maxdepth 1 -type l -delete; rmdir #{var_log_dir}"
-        end
+    # transfer existing contents to target directory
+    execute "rsync -a #{var_log_dir}/ #{real_log_dir}/" do
+        only_if { ::File.directory? var_log_dir }
     end
-    link var_log_dir do
-        to real_log_dir
+    directory "#{var_log_dir}" do
```

Comment:
should this have the same as line 82
 only_if { ::File.directory? var_log_dir }

Reply:
or should it be rmdir just in case it is the linked folder.

Reply:
`rm -rf` on a link to a directory removes only the link.